frontend/components/SlideShow.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="flex flex-col h-full" @keyup.left="left">
+    <div class="flex-grow" />
+    <div class="flex-0">
+      <div v-if="current==0">
+        <div class="text-6xl font-extrabold text-center"> 
+          <span>(#{{ model.pr.number }})</span>&nbsp;
+          <GradientText>{{ model.pr.title }}</GradientText>
+        </div>
+        <div class="mx-auto w-1/2 mt-8">
+          <Review-MetadataList :model="model" />
+        </div>
+      </div>
+
+      <div v-if="current===1">
+        <ComponentCard> 
+          <template #title>
+            <div class="text-xl font-extrabold">
+              <span>(#{{ model.pr.number }})</span>&nbsp;
+              <GradientText>{{ model.pr.title }}</GradientText>
+            </div>
+          </template>
+          <template #body>
+            <div class="px-4 py-4">
+              <BodyMarkdown :model="model" />
+            </div>
+          </template>
+        </ComponentCard>
+      </div>
+
+      <div v-for="(c, i) in model.comments" :key="i">
+        <SlideCard
+          v-if="i+2==current" :comment="c"
```

Comment:
This is kind of silly, but renders the `SlideCard` only if the current index (mapped/controlled) by `current` state and keypresses, is this one.

It's brittle because the number of slides preceding the comments is hardcoded to 2.

When we end up adding more slides either before the comments or after, this will break.